packages/nc-gui/composables/useViewData.ts

```diff
@@ -240,14 +240,20 @@ export function useViewData(
     }
   }
 
+  // inside this method use metaValue and viewMetaValue to refer meta
+  // since sometimes we need to pass old metas
   async function updateRowProperty(
     toUpdate: Row,
     property: string,
     { metaValue = meta.value, viewMetaValue = viewMeta.value }: { metaValue?: TableType; viewMetaValue?: ViewType } = {},
   ) {
     if (toUpdate.rowMeta) toUpdate.rowMeta.saving = true
+
+    // if the field name is missing return
+    if (!property) return
```

Comment:
The typing is inconsistent. If `property` can be nullish, it should be typed as such, i.e. `property?: string` or something.